backend/python/app/templates/oauth2-redirect.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html lang="en-US">
+<head>
+    <title>Swagger UI: OAuth2 Redirect</title>
+</head>
+<body>
+<script>
+    'use strict';
+    function run () {
+        var oauth2 = window.opener.swaggerUIRedirectOauth2;
+        var sentState = oauth2.state;
+        var redirectUrl = oauth2.redirectUrl;
+        var isValid, qp, arr;
+
+        if (/code|token|error/.test(window.location.hash)) {
```

Comment:
just curious: what does this if statement do (and what's up with the slashes, like /code)?

Reply:
I think this file was directly copied over from Swagger's repo so we don't need to look too deeply into it.

Looks like `window.location.hash` returns a part of the URL (see https://www.w3schools.com/jsref/prop_loc_hash.asp), the forward slashes are for denoting a regex pattern I believe